lib/index.js

```diff
@@ -72,6 +74,7 @@ exports.request = function (method, url, options, callback, _trace) {
     }
 
     var req = client.request(uri);
+    req.time = new Date().getTime();
```

Comment:
we need to find a safer place to store this data.  We can create a `now` var that contains this info and pass it into the log function as options.